src/scriptdev2/scripts/eastern_kingdoms/shadowfang_keep/shadowfang_keep.cpp

```diff
@@ -598,8 +598,11 @@ struct boss_arugalAI : public ScriptedAI
         {
             if (!m_bAttacking)
             {
-                DoCastSpellIfCan(m_creature->getVictim(), SPELL_VOID_BOLT);
-                m_uiVoidboltTimer = urand(2900, 4800);
+                if (Unit* victim = m_creature->getVictim())
```

Comment:
I have not spend a great deal on that one. It sounds like the whole script is maybe due for a workover making this commit even more redundant, I'm going to pull it from the lot since I don't think this commit is worth defending any further.